### DISCUSSION TITLE
table background

### DISCUSSION THREAD

**Discussion by ddkwork** [cogentcore/core#809] [Ideas]
After a single mouse or double-click event for each row of the table, i.e. when you click on the next row, we should reset the background color of the previously selected row

![{5893DCB9-3DF0-4261-A8A8-CBA64FB76644}](https://github.com/goki/goki/assets/19886504/05b6532b-e3f8-4a85-bf53-d8135792f1fc)
![{D57B5344-1B74-4946-9A7E-532EB974E924}](https://github.com/goki/goki/assets/19886504/daffe17a-c877-4fdf-bb76-0a029d30b289)

**Comment by kkoreilly** [comment]
I can not reproduce this issue in the latest version of goki. Can you pull the latest version of task manager and tell me if you still have the issue? Thanks!

**Comment by ddkwork** [comment]
> I can not reproduce this issue in the latest version of goki. Can you pull the latest version of task manager and tell me if you still have the issue? Thanks!

I pulled the latest commit, and there is still a problem, the bug reproduces the operation: click on the row table outside the row operation, try to scroll the mouse wheel

**Reply by kkoreilly** [reply]
I can reproduce the issue now and will work on solving it soon.
